UnitTests/Tests/FISAppDelegateSpec.m

```diff
@@ -0,0 +1,204 @@
+                                                     @"memoirs"
+                                                     ] mutableCopy]
+                           } mutableCopy],
+           @"Summer" : [@{ @"Independence Day" : [@[ @"fireworks"      ,
+                                                     @"barbeque"       ,
+                                                     @"picnic blanket" ,
+                                                     @"sunscreen"
+                                                     ] mutableCopy],
+                           @"Labor Day"        : [@[ @"white jeans"    ,
+                                                     @"shopping bag"
+                                                     ] mutableCopy]
+                           } mutableCopy],
+           @"Fall"   : [@{ @"Veteran's Day"    : [@[ @"barbeque"       ,
+                                                     @"American flag"
+                                                     ] mutableCopy],
+                           @"Thanksgiving Day" : [@[ @"turkey"         ,
+                                                     @"gravy"          ,
+                                                     @"mashed potatoes",
+                                                     @"acorn squash"   ,
+                                                     @"cranberry sauce",
+                                                     @"napkins"
+                                                     ] mutableCopy]
+                           } mutableCopy]
+           };
+
+
+    });
+    
+    describe(@"1 holidaysInSeason:", ^{
+        it(@"returns an array with the correct holidays in the winter season", ^{
+            expect([appDelegate holidaysInSeason:@"Winter"]).to.haveCountOf(2);
+            expect([appDelegate holidaysInSeason:@"Winter"]).to.contain(@"Christmas Day");
+            expect([appDelegate holidaysInSeason:@"Winter"]).to.contain(@"New Year's Day");
+        });
+        
+        it(@"returns an array with the correct holidays in the summer season", ^{
+            expect([appDelegate holidaysInSeason:@"Summer"]).to.haveCountOf(2);
+            expect([appDelegate holidaysInSeason:@"Summer"]).to.contain(@"Independence Day");
+            expect([appDelegate holidaysInSeason:@"Summer"]).to.contain(@"Labor Day");
+        });
+    });
+    
+    describe(@"2 suppliesInHoliday:inSeason:", ^{
+        it(@"returns an array of the correct supplies for New Year's Day in Winter ", ^{
+            NSMutableArray *newYearsSupplies = [@[ @"party hats",
+                                                   @"kazoos"    ,
+                                                   @"champagne"
+                                                   ] mutableCopy];
+            
+            expect([appDelegate suppliesInHoliday:@"New Year's Day"
+                                         inSeason:@"Winter"]).to.equal(newYearsSupplies);
```

Comment:
So I made [this cocoapod](https://github.com/jmburges/Expecta-Collections) to be able to equal in any order :)